images/gitlab-toolbox/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-task-runner-ee@sha256:c643675862fc5b3dac0a75329ddd12cba51fef2e50e5fbc69a5f14e1b7af0b92
+FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-toolbox-ee:v14.6.3
```

Comment:
What's the deal with this, I guess it's not called `task-runner` anymore?

Reply:
That's right, they renamed it to `gitlab-toolbox`.